build.gradle

```diff
@@ -0,0 +1,233 @@
+
+import net.ltgt.gradle.errorprone.CheckSeverity
+
+plugins {
```

Comment:
Gonna drop most of these plugins until we actually need them – really want to focus on not bloating the build (I've probably spent too much time on Go now :)).

Reply:
this sounds personal.  feel free to satisfy your OCD by removing whatever you feel is necessary.  i will usually just go with the flow